examples/basic_usage/README.md

```diff
@@ -1,4 +1,4 @@
-# Launch an example EC2 instance in a new VPC #
+# Example Terraform Code #
```

Comment:
Sentence case for titles I believe?

```suggestion
# Example Terraform code #
```

Reply:
[First level headings get title case](https://github.com/cisagov/development-guide/blob/develop/style-guide.md#titles-and-subjects).

Reply:
Ahh right. Sorry about that.